packages/devtools_app/test/shared/preferences_controller_test.dart

```diff
@@ -119,4 +124,67 @@ void main() {
       // preferences testable, then test it
     });
   });
+
+  group('$MemoryPreferencesController', () {
+    late MemoryPreferencesController controller;
+    late FlutterTestStorage storage;
```

Comment:
we shouldn't need to store this var locally since we are setting a global. We should just be able to access it via the global var `storage`